node_modules/oae-jitsi/lib/activity.js

```diff
@@ -0,0 +1,553 @@
+ */
+var _meetingMessageProducer = function(resource, callback) {
+    var meetingId = resource.resourceData.meetingId;
+    var message = resource.resourceData.message;
+    MeetingsDAO.getMeeting(meetingId, function(err, meeting) {
+        if (err) {
+            return callback(err);
+        }
+
+        MessageBoxUtil.createPersistentMessageActivityEntity(message, function(err, entity) {
+            if (err) {
+                return callback(err);
+            }
+
+            // Store the meeting id and visibility on the entity as these are required for routing the activities
+            entity.objectType = 'meeting-jitsi-message';
+            entity.meetingId = meeting.id;
+            entity.meetingVisibility = meeting.visibility;
+            return callback(null, entity);
+        });
+    });
+};
+
+/**
+ * Create the persistent meeting entity that can be transformed into an activity entity for the UI.
+ *
+ * @param  {Meeting}     meeting      The meeting that provides the data for the entity.
+ * @return {Object}                         An object containing the entity data that can be transformed into a UI meeting activity entity
+ * @api private
+ */
+var _createPersistentMeetingActivityEntity = function(meeting) {
+    return new ActivityModel.ActivityEntity('meeting-jitsi', meeting.id, meeting.visibility, {'meeting-jitsi': meeting});
+};
+
+/*
+ * Transform the meeting persistent activity entities into UI-friendly ones
+ * @see ActivityAPI#registerActivityEntityType
+ */
+var _meetingTransformer = function(ctx, activityEntities, callback) {
+
+    var transformedActivityEntities = {};
+
+    var allRevisionIds = [];
+    _.each(activityEntities, function(entities, activityId) {
+        transformedActivityEntities[activityId] = transformedActivityEntities[activityId] || {};
+
+        _.each(entities, function(entity, entityId) {
+            // Transform the persistent entity into an ActivityStrea.ms compliant format
+            transformedActivityEntities[activityId][entityId] = _transformPersistentMeetingActivityEntity(ctx, entity);
+        });
+    });
+
+    return callback(null, transformedActivityEntities);
+
+};
+
+/*
+ * Transform the persisted message activity entities into UI-friendly ones
+ * @see ActivityAPI#registerActivityEntityType
+ */
+var _meetingMessageTransformer = function(ctx, activityEntities, callback) {
+    var transformedActivityEntities = {};
+    _.keys(activityEntities).forEach(function(activityId) {
+        transformedActivityEntities[activityId] = transformedActivityEntities[activityId] || {};
+        _.keys(activityEntities[activityId]).forEach(function(entityId) {
+            var entity = activityEntities[activityId][entityId];
+            var meetingId = entity.meetingId;
+            var resource = AuthzUtil.getResourceFromId(meetingId);
+            var profilePath = '/meeting-jitsi/' + resource.tenantAlias + '/' + resource.resourceId;
```

Comment:
For this line it would be `util.format('/meeting-jitsi/%s/%s', resource.tenantAlias, resource.resourceId)` (so your string with %s as the placeholder param, then the params you want to use). You'll also need to add `var util = require('util')`.